UndertaleModTool/Converters/UndertaleCachedImageLoader.cs

```diff
@@ -281,7 +284,7 @@ public object Convert(object[] values, Type targetType, object parameter, Cultur
             if (values[0] is null) // tile
                 return null;
 
-            if ((uint)values[3] == 0 || (uint)values[4] == 0) // width, height
+            if ((uint)values[1] == 0 || (uint)values[2] == 0) // width, height
```

Comment:
Why were these changed? Whats the deal behind those

Reply:
It's related to the tile update fix _(see changes starting from line 1126 of "UndertaleRoomEditor.xaml")_.